SimG4CMS/Calo/test/python/minbias2021_cfg.py

```diff
@@ -0,0 +1,111 @@
+###############################################################################
+# Way to use this:
+#   cmsRun minbias2021_cfg.py type=DDD var=ZeroMaterial
+#
+#   Options for type: DDD, DD4hep
+#           for var: ZeroMaterial, FlatMinus05Percent, FlatMinus10Percent,
+#                    FlatPlus05Percent, FlatPlus10Percent
+#
+###############################################################################
+import FWCore.ParameterSet.Config as cms
+import os, sys, importlib, re, random
+import FWCore.ParameterSet.VarParsing as VarParsing
+
+####################################################################
+### SETUP OPTIONS
+options = VarParsing.VarParsing('standard')
```

Comment:
Informational comment: while VarParsing is still allowed, it should be considered deprecated and argparse should be preferred, since it is now supported by cmsRun (since #42650). An example migration can be found here: https://github.com/cms-sw/cmssw/commit/ec33ffd33f08eb3de847a56b5aff616271b57a61